Sources/Components/Klimabrolet/Views/KlimabroletActionsView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import Foundation
+
+protocol KlimabroletActionsViewDelegate: AnyObject {
+    func klimabroletViewDidSelectPrimaryButton(_ view: KlimabroletActionsView)
+    func klimabroletViewDidSelectSecondaryButton(_ view: KlimabroletActionsView)
+}
+
+class KlimabroletActionsView: UIView {
+    public var delegate: KlimabroletActionsViewDelegate?
+
+    // MARK: - Private subviews
+
+    private(set) lazy var primaryButton: UIButton = {
+        let button = Button(style: .callToAction)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.numberOfLines = 0
+        button.addTarget(self, action: #selector(handleTapOnPrimaryButton), for: .touchUpInside)
+        return button
+    }()
+
+    private(set) lazy var secondaryButton: UIButton = {
```

Comment:
Do we need to expose these two buttons, or could the be `private`?

Reply:
Ah, I see now why they're `private(set)`.

Could you instead make a method called `configure(primaryButtonTitle: String, secondaryButtonTitle: String)` to use from the `KlimabroletView.configure(…)` method?